scripts/validate_docstrings.py

```diff
@@ -399,52 +409,60 @@ def header(title, width=80, char="#") -> str:
         sys.stderr.write(header("Doctests"))
         sys.stderr.write(result["examples_errs"])
 
+    return len(result["errors"]) + len(result["examples_errs"])
+
+
+def _format_ignore_errors(raw_ignore_errors):
+    ignore_errors = collections.defaultdict(set)
+    if raw_ignore_errors:
+        for obj_name, error_codes in raw_ignore_errors:
+            # function errors "pandas.Series PR01,SA01"
+            if obj_name != "*":
```

Comment:
Could we just use a separate flag for ignoring all errors for a specific code?

Reply:
I think it's simpler this way. In a follow up PR I'll try to remove the star. So, we'll be able to simply use `--ignore-errors PR01`. I guess what you don't like is the star?

And I may use `None` for the key when the error should always be ignored. But since this PR became already too big, I preferred not to also edit the `argparse` here.

What do you think?

Reply:
Yeah the `*`, while commonplace in other tools, is still a little more opaque to me than `--ignore-all CODE` or similar.

Not a blocker to me, but would be nice to consider in a followup